stdlib/public/stubs/CommandLine.cpp

```diff
@@ -242,16 +244,50 @@ char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
 
   return argv;
 }
+#elif defined(__OpenBSD__)
+#include <sys/exec.h>
+#include <sys/sysctl.h>
+#include <sys/types.h>
+
+SWIFT_RUNTIME_STDLIB_API
+char **_swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
+  assert(outArgLen != nullptr);
+  if (_swift_stdlib_ProcessOverrideUnsafeArgv) {
+    *outArgLen = _swift_stdlib_ProcessOverrideUnsafeArgc;
+    return _swift_stdlib_ProcessOverrideUnsafeArgv;
+  }
+
+  int mib[2] = {CTL_VM, VM_PSSTRINGS};
+  struct _ps_strings _ps;
+  size_t len = sizeof(_ps);
+
+  if (sysctl(mib, 2, &_ps, &len, NULL, 0) == -1) {
+    char **empty_argv = static_cast<char **>(calloc(1, sizeof(char *)));
+    empty_argv[0] = nullptr;
+    return empty_argv;
```

Comment:
You need to write 0 into `*outArgLen` here.

Reply:
Ugh, sorry about that.